azure-pipelines-full-tests.yml

```diff
@@ -107,8 +107,8 @@ jobs:
       fullTest: true
       jobName: 'FullTest'
 
-- job: AutomationFullTestPython310ProfileLatest
-  displayName: Automation Full Test Python310 Profile Latest
+- job: AutomationFullTestPython311ProfileLatest
```

Comment:
I don't think we should drop Python 3.10 tests right away as the bundled Python is still 3.10.

Reply:
Bundled Python is also bumped in https://github.com/Azure/azure-cli/pull/26749/